cvat-ui/src/reducers/interfaces.ts

```diff
@@ -21,6 +21,37 @@ export interface AuthState {
     allowResetPassword: boolean;
 }
 
+export interface ProjectsQuery {
+    page: number;
+    id: number | null;
+    search: string | null;
+    owner: string | null;
+    name: string | null;
+    status: string | null;
+    [key: string]: string | number | null | undefined;
+}
+
+export interface Project {
+    instance: any; // cvat-core instance
+}
```

Comment:
Do we really need this additional instance with a single field? Such the instance has been created for Task because there was a preview also.